src/org/zaproxy/zap/extension/websocket/WebSocketAPI.java

```diff
@@ -431,6 +438,24 @@ public ApiResponse handleApiView(String name, JSONObject params) throws ApiExcep
                 LOG.error(e.getMessage(), e);
                 throw new ApiException(ApiException.Type.INTERNAL_ERROR);
             }
+        } else if (VIEW_BREAK_TEXT_MESSAGE.equals(name)) {
+            ExtensionBreak extBreak = Control.getSingleton().getExtensionLoader().getExtension(ExtensionBreak.class);
+            if (extBreak != null) {
+                Message msg = extBreak.getBreakpointManagementInterface().getMessage();

```

Comment:
Could extract a method, that returns the (`WebSocketMessageDTO`) message or throws.